internal/pkg/describe/uri.go

```diff
@@ -409,7 +409,7 @@ func (u *accessURI) strings() []string {
 			protocol = "https://"
 		}
 		path := ""
-		if u.Path != "/" {
+		if !strings.HasPrefix(Path, "/") {
 			path = fmt.Sprintf("/%s", u.Path)
 		}
 		uris = append(uris, color.HighlightResource(protocol+dnsName+path))
```

Comment:
If the user adds a path without the `/` prefix, the `path` variable here will still be `""`. I think we need to catch this case. Thanks!